.github/workflows/test.yml

```diff
@@ -178,7 +183,7 @@ jobs:
 
       - name: SonarCloud Scan
         if: github.actor != 'dependabot[bot]' && github.actor != 'dependabot-preview[bot]'
-        uses: SonarSource/sonarcloud-github-action@v1.6
+        uses: SonarSource/sonarcloud-github-action@v2.0.1
```

Comment:
According to the docs we should consider switching this to the SonarQube Plugin for Gradle.

Reply:
That is the sonar action for visualization, visualization is not managed by gradle, so this is the correct way.